backend/src/main/kotlin/com/tngtech/apicenter/backend/config/RestResponseExceptionHandler.kt

```diff
@@ -51,5 +51,14 @@ class RestResponseExceptionHandler {
     fun handleServiceIdMismatch(exception: MismatchedSpecificationIdException) =
             makeResponseEntity("The API ID used in the upload URL (${exception.urlPathId}) " +
                     "is not the same as the API ID used in the specification body (${exception.userDefinedId})", HttpStatus.BAD_REQUEST)
+
+    @ExceptionHandler(IdNotCastableToLongException::class)
+    fun handleUncastableId(exception: IdNotCastableToLongException) =
+            makeResponseEntity("A specifications must have an ID that can be parsed to a Java Long type" +
```

Comment:
This is going to be fixed with a later Spring Security version, right? Then I'd rather wait until it is fixed before we bake some workarounds into our logic.